Packs/Druva/Integrations/Druva/Druva.py

```diff
@@ -0,0 +1,358 @@
+    statusCode = response.status_code
+    if (statusCode == 200):
+        responseJson = response.json()
+        headers = ['RangeID']
+        readable_output = tableToMarkdown('Quarantine Range Deleted Successfully', str(range_id), headers=headers)
+        raw_response = responseJson
+        outputs = {"Druva.deletedQuarantineRange(val.rangeID == obj.rangeID)": str(range_id)}
+        return (readable_output, outputs, raw_response)
+    else:
+        demisto.log('Error: ' + str(response.status_code))
+        return None
+
+
+def Druva_ViewQurantineRange_Command(clientObj, resource_id, range_id):
+    response = clientObj.view_quarantineRange(resource_id, range_id)
+    statusCode = response.status_code
+    if (statusCode == 200):
+        responseJson = response.json()
+        readable_output = tableToMarkdown('Range Details', responseJson)
+        outputs = {"Druva.viewedQuarantineRange(val.rangeID == obj.rangeID)": responseJson}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+
+
+def Druva_UpdateQuarantineRange_Command(clientObj, resource_id, resource_type, range_id, from_date, to_date):
+    response = clientObj.update_quarantineRange(resource_id, resource_type, range_id, from_date, to_date)
+    statusCode = response.status_code
+    if (statusCode == 200):
+        responseJson = response.json()
+        headers = ['RangeID']
+        readable_output = tableToMarkdown('Range updated successfully', str(responseJson['rangeID']), headers=headers)
+        outputs = {"Druva.updatedQuarantineRange": str(range_id)}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+    else:
+        demisto.log('Error: ' + str(response.status_code))
+        return None
+
+
+def Druva_ListQuarantine_Snapshots_Command(clientObj, resource_id, range_id):
+    response = clientObj.get_quarantinedSnapshots(resource_id, range_id)
+    statusCode = response.status_code
+    if (statusCode == 200):
+        responseJson = response.json()
+        readable_output = tableToMarkdown('Quarantined Snapshots', responseJson['snapshots'])
+        outputs = {"Druva.quarantinedSnapshots(val.snapshotID == obj.snapshotID)": responseJson['snapshots']}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+    else:
+        demisto.log('Error: ' + str(response.status_code))
+        return None
+
+
+def Druva_DeleteQuarantined_Snapshots_Command(clientObj, resource_id, range_id, snapshot_id):
+    response = clientObj.delete_Snapshots(resource_id, range_id, snapshot_id)
+    statusCode = response.status_code
+    if (statusCode == 200):
+        responseJson = response.json()
+        headers = ['Snapshot ID']
+        readable_output = tableToMarkdown('Snapshot Deleted successfully', str(snapshot_id), headers=headers)
+        outputs = {"Druva.deletedSnapshots(val.snapshotID == obj.snapshotID)": str(snapshot_id)}
```

Comment:
delete should not return context outputs

Reply:
Requested Changes made.